openedx/core/djangoapps/user_api/preferences/tests/test_api.py

```diff
@@ -191,16 +191,16 @@ def test_update_user_preferences_errors(self, user_preference_save, user_prefere
             self.test_preference_key: "new_value"
         }
         with self.assertRaises(UserNotFound):
-            update_user_preferences(self.user, update_data, username="no_such_user")
+            update_user_preferences(self.user, update_data, user="no_such_user")
 
         with self.assertRaises(UserNotFound):
             update_user_preferences(self.no_such_user, update_data)
 
         with self.assertRaises(UserNotAuthorized):
-            update_user_preferences(self.staff_user, update_data, username=self.user.username)
+            update_user_preferences(self.staff_user, update_data, user=self.user.username)
 
         with self.assertRaises(UserNotAuthorized):
-            update_user_preferences(self.different_user, update_data, username=self.user.username)
+            update_user_preferences(self.different_user, update_data, user=self.user.username)
```

Comment:
Please add some smoke testing for when a User is passed in instead of a username.